user/caching.md

```diff
@@ -134,17 +134,27 @@ caches `$HOME/.cache/pip`.
 
 ### ccache cache
 
-For caching `ccache` files, use:
+If you are using `ccache`, use:
 
 ```yaml
 language: c # or other C/C++ variants
 
 cache: ccache
-```
+``
```

Comment:
Oops! Missing quote here! :) and I think that the `automatically` of the next line is also duplicated, right?